Sources/SwiftDocC/Infrastructure/Link Resolution/PathHierarchy.swift

```diff
@@ -864,6 +864,9 @@ private extension PathHierarchy.Node {
     func fullNameOfValue(context: DocumentationContext) -> String {
         guard let identifier = identifier else { return name }
         if let symbol = symbol {
+            if let fragments = symbol[mixin: SymbolGraph.Symbol.DeclarationFragments.self]?.declarationFragments {
+                return fragments.map { $0.spelling }.joined().split(whereSeparator: { $0.isWhitespace || $0.isNewline }).joined(separator: " ")
```

Comment:
A small suggestion: Could we remove `whereSeparator` label? It seems a litter inconsistant.
```suggestion
                return fragments.map { $0.spelling }.joined().split { $0.isWhitespace || $0.isNewline }.joined(separator: " ")
```

Reply:
I personally find that the argument label for `split` makes it easier for me to contextualize the closure when reading it. I don't mind the surrounding parenthesis being inconsistent but if I were to make it consistent my preference would be 
```suggestion
                return fragments.map({ $0.spelling }).joined().split(whereSeparator: { $0.isWhitespace || $0.isNewline }).joined(separator: " ")
```

Reply:
I agree about the argument label `fragments.map(\.spelling).joined().split(whereSeparator: { $0.isWhitespace || $0.isNewline }).joined(separator: " ")`